saleor/static/js/components/variantPicker/VariantPicker.js

```diff
@@ -60,13 +72,44 @@ export default class VariantPicker extends Component {
       selection: Object.assign({}, this.state.selection, { [attrId]: valueId })
     }, () => {
       this.matchVariantFromSelection();
+      let params = {};
+      Object.keys(this.state.selection).forEach(attrId => {
+        const attribute = this.matchAttribute(attrId);
+        const value = this.matchAttributeValue(attribute, this.state.selection[attrId]);
+        if (attribute && value) {
+           params[attribute.name] = value.slug;
+        }
+      });
+      history.pushState(null, null, '?'+ queryString.stringify(params));
```

Comment:
Missing space before `+` operator. I'd recommend using [ESLint](http://eslint.org/) when developing JS code. There is `eslintrc.json` in the repository which defines set of common rules that we follow to keep our code formatting consistent.